### DISCUSSION TITLE
JSON-LD response does not conform to JSON-LD 1.1 spec

### DISCUSSION THREAD

**Discussion by sjvanrossum** [weather-gov/api#316] [Q&A]
**Describe the bug**
JSON-LD API responses specify `@context`'s `@version` property as `"@context": "1.1"`, but the spec for JSON-LD 1.1 states that this field must be a number, mandating that it be returned as `"@version": 1.1`. In addition to being malformed JSON-LD 1.1 this currently breaks compatibility with [digitalbazaar/pyld](https://github.com/digitalbazaar/pyld) and [digitalbazaar/jsonld.js](https://github.com/digitalbazaar/jsonld.js).

**To Reproduce**
https://api.weather.gov/stations?state=AL&limit=1
https://api.weather.gov/stations/C1231
https://api.weather.gov/zones/forecast/ALZ266
https://api.weather.gov/zones/county/ALC003
https://api.weather.gov/zones/fire/ALZ266
https://api.weather.gov/stations/C1231/observations/latest

**Expected behavior**
"If the [context definition](https://www.w3.org/TR/json-ld11/#dfn-context-definition) has an `@version` key, its value _MUST_ be a [number](https://tc39.es/ecma262/#sec-terms-and-definitions-number-value) with the value `1.1`." ([JSON-LD 1.1, § 9.15 Context Definitions](https://www.w3.org/TR/json-ld11/#context-definitions))
"The unaliased `@version` keyword _MAY_ be used as a key in a [context definition](https://www.w3.org/TR/json-ld11/#dfn-context-definition). Its value _MUST_ be a [number](https://tc39.es/ecma262/#sec-terms-and-definitions-number-value) with the value `1.1`. This keyword is described further in [§ 9.15 Context Definitions](https://www.w3.org/TR/json-ld11/#context-definitions)." ([JSON-LD 1.1, § 9.16 Keywords](https://www.w3.org/TR/json-ld11/#keywords))

Expected behavior would be that either `"@version": 1.1` is sent or `@version` is not sent at all, since it is optional.

**Environment**
Python requests, Chrome XHR, browser response.

Chrome console (replace URL, likely prints `string` upon completion): 
```javascript
fetch('https://api.weather.gov/path/to/resource', {
    method: 'GET',
    headers: {
      'Accept': 'application/ld+json'
    }
  })
  .then(res => res.json())
  .then(json => console.log(typeof json['@context']['@version']))
```

**Comment by StephenClouse** [comment]
We will look into fixing this.

However, it should be noted that our current intent is to deprecate JSON-LD support at a later date. We would encourage using the GeoJSON output and disregarding the JSON-LD components.

**Comment by sjvanrossum** [comment]
Thank you! I'll have a look at how I can incorporate GeoJSON instead of JSON-LD.
